app/services/idv/steps/document_capture_step.rb

```diff
@@ -23,6 +23,7 @@ def extra_view_variables
         url_builder = ImageUploadPresignedUrlGenerator.new
 
         {
+          document_capture_session_uuid: flow_session[:document_capture_session_uuid],
```

Comment:
(I reworked the underlying view to require `document_capture_session_uuid`)

Reply:
I think we need to make this change to DocumentCaptureStep and the view in a separate PR that gets merged ahead of this PR. And think carefully about the 50/50 state for that. Maybe we need to add the parameter in one deployment, and then change the view in the next one.